src/utils.js

```diff
@@ -0,0 +1,59 @@
+export function toArray(nodes) {
+	// SCRIPT5014 in IE8
+	const array = [];
+
+	if (nodes) {
+		for (let i = 0, len = nodes.length; i < len; i++) {
+			array.push(nodes[i]);
+		}
+	}
+	return array;
+}
+
+/**
+ * Select or create element
+ * @param {String|HTMLElement|jQuery} param
+ *  when string given is as HTML tag, then create element
+ *  otherwise it returns selected elements
+ * @param {Boolean} multi
+ * @returns {HTMLElement}
```

Comment:
@sculove Need to say it returns Array of HTMLElement

Reply:
ok thanks.